githubapp/middleware.go

```diff
@@ -28,6 +29,8 @@ const (
 	MetricsKeyRequests3xx = "github.requests.3xx"
 	MetricsKeyRequests4xx = "github.requests.4xx"
 	MetricsKeyRequests5xx = "github.requests.5xx"
+
+	MetricsKeyRequestsCached = "github.requests.cached"
```

Comment:
We could also introduce additional metrics to correspond with github rate limits for the given app using gauges.